h/static/styles/site.scss

```diff
@@ -1,40 +1,52 @@
-@import 'partials/base';
-@import 'partials/reset';
-@import 'partials/elements';
-@import 'partials/util';
+// Hypothesis Main
+// ============================================================================
+
+// Core
+// ----------------------------------------------------------------------------
+
+@import 'core/reset';
+@import 'core/color';
+@import 'core/typography';
+@import 'core/layout';
+@import 'core/icons';
+@import 'core/utilities';
 
 // Components
-@import 'partials/api-token';
-@import 'partials/annotation-card';
-@import 'partials/brand';
-@import 'partials/btn';
-@import 'partials/dropdown-menu';
-@import 'partials/footer';
-@import 'partials/form-actions';
-@import 'partials/form-checkbox';
-@import 'partials/form-container';
-@import 'partials/form-input';
-@import 'partials/form';
-@import 'partials/group-invite';
-@import 'partials/join-group-form';
-@import 'partials/link';
-@import 'partials/list-input';
-@import 'partials/lozenge';
-@import 'partials/masthead';
-@import 'partials/nav-bar';
-@import 'partials/pager';
-@import 'partials/search';
-@import 'partials/search-bar';
-@import 'partials/search-result-sidebar';
-@import 'partials/share-widget';
-@import 'partials/svg-icon';
-@import 'partials/tooltip';
-@import 'partials/tabs';
-@import 'partials/text';
+// ----------------------------------------------------------------------------
+
+@import 'components/api-token';
+@import 'components/annotation-card';
+@import 'components/brand';
+@import 'components/btn';
+@import 'components/dropdown-menu';
+@import 'components/footer';
+@import 'components/form';
+@import 'components/form-actions';
+@import 'components/form-checkbox';
+@import 'components/form-container';
+@import 'components/form-input';
+@import 'components/group-invite';
+@import 'components/join-group-form';
+@import 'components/link';
+@import 'components/list-input';
+@import 'components/lozenge';
+@import 'components/masthead';
+@import 'components/nav-bar';
+@import 'components/pager';
+@import 'components/search';
+@import 'components/search-bar';
+@import 'components/search-result-sidebar';
+@import 'components/share-widget';
+@import 'components/tooltip';
+@import 'components/tabs';
 
+// Rules
+// ----------------------------------------------------------------------------
+
+// This rule causes adverse effects in the admin and help pages, so it's
+// included here until we can tackle typography generally
```

Comment:
What adverse effects specifically?

Reply:
Ah - to explain, I considered "adverse effects" here to mean "changes the appearance of the page," since this PR is just about moving things around. But specifically, `font-normal` makes the text smaller and the line-height more compact on both the admin and help pages.

> "until we can tackle typography generally"
This refers to the bigger effort to move away from `px` generally, but also to adjust headings, spacing, etc.